src/core/agent/records/credentialMetadataRecord.types.ts

```diff
@@ -1,6 +1,7 @@
 enum CredentialMetadataRecordStatus {
   CONFIRMED = "confirmed",
   PENDING = "pending",
+  REVOKED = "revoked",
```

Comment:
hmm maybe we can refactor things and delete `CredentialMetadataRecordStatus` and just use `CredentialStatus` enum instead? they have the same string values so I don't think there's any need for migrations